docker/Makefile

```
@@ -1,6 +1,6 @@
-TAG=$(shell git describe)
+TAG=$(shell git describe --always)
 LABEL_NAMESPACE=uk.ac.sanger
-TAG_NAMESPACE=wsinpg
+TAG_NAMESPACE=ghcr.io/$(REPO_OWNER)
```

How about 
```suggestion
ifdef REPO_OWNER
TAG_NAMESPACE?=ghcr.io/$(REPO_OWNER)
else
TAG_NAMESPACE?=wsinpg
endif
```
allow us to run with a `TAG_NAMESPACE` suitable for DockerHub, or of choice with `make TAG_NAMESPACE=foo all`